tests/statement_serializer_tests/select_constraints.cpp

```diff
@@ -71,6 +71,20 @@ TEST_CASE("statement_serializer select constraints") {
             value = serialize(expression, context);
             expected = R"(FROM "users" "u")";
         }
+#ifdef SQLITE_ORM_WITH_CPP20_ALIASES
+        {
+            SECTION("with alias 2") {
+                auto expression = from<alias_<'u'>.for_<User>()>();
```

Comment:
why don't we use `alias<'u'>` instead of `alias_<'u'>`?

Reply:
Because I thought it resembles the current predefined table aliases. `alias_<'u'>` resembles `alias_u`.
However, the question is whether we want to have this syntax in the library at all.

The other available option is to use `"u"_alias`:
1. `constexpr auto u = "u"_alias.for_<User>()`
2. `constexpr auto u = alias_<'u'>.for_<User>()` - yes/no?

Reply:
`constexpr auto u = "u"_alias.for_<User>()` - nice
`constexpr auto u = alias<'u'>.for_<User>()` also but without redundant underscore as I wrote it. Underscore is a service symbols and shouldn't be used if it is not required. If you see that `alias_<'u'>` replaces `alias_u` then you can see it like underscore is replaced with templates brackets